speechbrain/dataio/sampler.py

```diff
@@ -10,6 +10,7 @@
 """
 import torch
```

Comment:
Ok, could you elaborate on why this is necessary? I remember the issue, but at the same time, we have something that wraps the sampler automatically to prevent this to happen, no ? In which use cases our wrapping isn't sufficient ?